docs/guide/features.md

```diff
@@ -368,6 +370,38 @@ By default, the worker script will be emitted as a separate chunk in the product
 import MyWorker from './worker?worker&inline'
 ```
 
+### new Worker()
+
+A web worker script can be imported by `new Worker()`. The difference between this syntax and imported by appending `?worker` or `?sharedworker` is that this one can use worker options. So we can create a `classic worker` by this syntax.
+
+Detect the case where `{ type: 'classic' }` is not provided or `{ type: 'classic' }` is passed with a static string and use classic workers.
+
+```ts
+const worker = new Worker(new URL('./worker.js', import.meta.url), {
+  type: 'classic'
+})
+const worker = new SharedWorker(new URL('./worker.js', import.meta.url), {
+  type: 'classic'
+})
+```
```

Comment:
We can simplify this part and explain `type: classic` later. Note: Github is not liking the formatting below.
```suggestion
```ts
const worker = new Worker(new URL('./worker.js', import.meta.url))
```\
```